stdlib/math.gr

```diff
@@ -0,0 +1,71 @@
+/* grainc-flags --compilation-mode=runtime */
+import WasmI32 from "runtime/unsafe/wasmi32"
+import WasmI64 from "runtime/unsafe/wasmi64"
+import WasmF64 from "runtime/unsafe/wasmf64"
+import {
+  (+),
+  (-),
+  (/),
+  (*),
+  (>),
+  coerceFloat64,
+  reducedInteger
+} from "runtime/numbers"
+import { newFloat64 } from "runtime/dataStructures"
+
+primitive (||) : (Bool, Bool) -> Bool = "@or"
+primitive throw : Exception -> a = "@throw"
+
+exception UnknownNumberTag
+
+// add
+export let add = (+)
+// sub
+export let sub = (-)
+// mul
+export let mul = (*)
+// div
+export let div = (/)
+// sqrt
+export let sqrt = (x: Number) => {
+  let x = WasmI32.fromGrain(x)
+  let xval = coerceFloat64(x)
+  let sqrtd = newFloat64(WasmF64.sqrt(xval))
+  WasmI32.toGrain(sqrtd): Number
+}
+// min
+export let min = (x: Number, y: Number) => if (y > x) x else y
+// max
+export let max = (x: Number, y: Number) => if (x > y) x else y
+// ceil
+export let ceil = (x: Number) => {
```

Comment:
For any of the functions that use the unsafe wasm values, can you add the `@disableGC` attribute? Here's an example: https://github.com/grain-lang/grain/blob/main/stdlib/bytes.gr#L59